### ISSUE TITLE
Global config (~/.config/tab.yml) is not properly handled when cwd is ~/.config or its subdirectory

### ISSUE THREAD

**Issue by solarizedalias** [austinjones/tab-rs#309]
## problem
It looks like the global config (~/.config/tab.yml) is also recognized as a local config to ~/.config when cwd is ~/.config or any of its subdirectory. 
And if you have a tab configured with a relative path from home directory (which works fine as a global config), that causes errors as the path doesn't make sense as a local config to ~/.config.

Also it says I have duplicate definitions, which is why I believe it's recognized both as a local config and the global config. This happens no matter if the path is configured relative or absolute.

## step to reproduce

OS: macOS Mojave 10.14.6 18G7016 x86_64
Shell: zsh 5.8
tab: tab 0.5.5

```

$ cat ~/.config/tab.yml
shell: zsh
workspace:
  - tab: N
    doc: N
    dir: solarizedalias/N # This is relative from $HOME ($HOME/solarizedalias/N)

  - tab: localbin
    doc: /usr/local/bin
    dir: /usr/local/bin # absolute path

  - tab: dotfiles
    doc: dotfiles
    dir: dotfiles # This is also a relative path

$ cd ~/.config
$ tab --check

5 errors were detected:
    - Tab N/ has a working directory that does not exist: /Users/ykksd/.config/solarizedalias/N
    - Tab dotfiles/ has a working directory that does not exist: /Users/ykksd/.config/dotfiles
    - Tab N/ has duplicate definitions
    - Tab localbin/ has duplicate definitions
    - Tab dotfiles/ has duplicate definitions
```

## expected behavior
~/.config/tab.yml is treated as the global config
(But I wonder what if someone really wants a local config at ~/.config?)

## By the way...
I really like tab! I see it's improving at every release. Good work!

**Comment by austinjones** [comment]
Hi @solarizedalias,
Thanks for using tab!  That is really cool to hear :D

Indeed, it looks like the config is being scanned twice.  There is some 'working directory' magic that is applied to the global config, and that's why the relative paths are being rendered incorrectly.

I'll fix this by ignoring the global config if it appears in the scan.

**Comment by austinjones** [comment]
I just published a fix in [v0.5.6](https://github.com/austinjones/tab-rs/releases/tag/v0.5.6).
